src/clients/node/ci.zig

```diff
@@ -11,13 +11,15 @@ const TmpTigerBeetle = @import("../../testing/tmp_tigerbeetle.zig");
 pub fn tests(shell: *Shell, gpa: std.mem.Allocator) !void {
     assert(shell.file_exists("package.json"));
 
-    // We have some unit-tests for node, but they are likely bitrotted, as they are not run on CI.
-
     // Integration tests.
 
     // We need to build the tigerbeetle-node library manually for samples to be able to pick it up.
     try shell.exec("npm install", .{});
 
+    // Run node client unit tests and benchmark after installing tigerbeetle-node
+    try shell.exec("npm run test", .{});
+    try shell.exec("npm run benchmark", .{});
```

Comment:
How long does it take to run the benches? These tests gate the main branch, so they must be very fast (granted, currently our overall test suite isn't nearly fast enough, but the direction we want to go is "faster tests"). 

Usually, the benchmarks are quite slow, so they are removed from the CI, _or_ are being run with artificially low iteration counts, so that we don't _actually_ benchmark anything, but just smoke test that there are no errors. 

That is to say: if `nmp run benchmark` is significatnly slower than just `npm run test`, we need to do something here. If it's fast enough as is, than that's fine already!

Reply:
`test` and `benchmark` are effectively instant (the latter only makes `10` transfers with batch size `5120`, so acts more like a smoke-test on throughput)